### DISCUSSION TITLE
:pray: Easy way to propagate scroll event on gesture container?!

### DISCUSSION THREAD

**Discussion by GeorgeHop** [software-mansion/react-native-gesture-handler#2316] [General]
HI to all! I alredy spended two days to find best and simple solution to achieve this animation

https://user-images.githubusercontent.com/47904385/201159920-a0c55b05-bc46-4888-bc23-03cb23f49d91.mp4

So after opening gesture container fully, scrollview start responding and then when you scroll to top gesture container will start working againg and. So I want to achive animation like on the bottom video.

https://user-images.githubusercontent.com/47904385/201161032-9d0da8fc-3b0e-4cc8-9fd2-aaabb92eaba1.mp4

I found the lib. https://github.com/rgommezz/react-native-scroll-bottom-sheet/blob/master/src/index.tsx where guys making it in some very tricky way I think should be some easier and better way

**Comment by dorkyboi** [comment]
I am also interested in how this could be implemented. We need this in a project

**Comment by ChrisYohann** [comment]
Why don't you use [react-native-bottom-sheet](https://github.com/gorhom/react-native-bottom-sheet) for this ? It has already List components like `BottomSheetScrollView` or `BottomSheetFlatList` which are wrapper around native components, but enhanced to allow the gesture you want : basically they are locked until the bottom sheet is fully extended, and once you are to the top it starts scrolling.

**Reply by GeorgeHop** [reply]
It don't work with scroll inside. And they use very strange solution for that. I already found how to achieve animation like that without over-engineering.

**Reply by GeorgeHop** [reply]
You already have all tools to create such animation... Why use external lib which already not supported:) 

Link to thread with example code for modal. And also I have done animation from my video. 

Link: https://github.com/software-mansion/react-native-gesture-handler/discussions/2222

**Reply by pioner92** [reply]
@GeorgeHop hi
How did you implement this ?
